sdk/python/foundation-models/benchmarking/evaluating_claude_models.ipynb

```diff
@@ -0,0 +1,205 @@
+    "- Access keys for Claude endpoints on [Amazon Web Services Bedrock platform](https://aws.amazon.com/bedrock/claude/)  \n",
+    "\n",
+    "## Configuring a Workspace connection for Bedrock access\n",
+    "You will use a Workspace connection to securely store Bedrock access keys. Follow the steps below to create a custom-type connection:\n",
+    "- Follow directions for [creating a custom connection in the AzureML studio UI](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/tools-reference/python-tool?view=azureml-api-2#create-a-custom-connection)\n",
+    "- Add the following two key-value pairs to the custom connection:\n",
+    "  1. A key named `AccessKey` with a value containing your AWS access key\n",
+    "  2. A key named `SecretKey` with a value containing your AWS secret access key \n",
+    "\n",
+    "## Configuring and running an evaluation pipeline\n",
+    "Please set global values in the following cell for your AzureML Workspace, the Bedrock endpoint you want to call, the name of connection you created in the previous step, and the name of the eval you want to run.\n",
+    "\n",
+    "Supported evals are the following: `boolq`, `gsm8k`, `hellaswag`, `human_eval`, `mmlu_humanities`, `mmlu_other`, `mmlu_social_sciences`, `mmlu_stem`, `openbookqa`, `piqa`, `social_iqa`, `winogrande`.\n",
+    "\n",
+    "You can also set the sample ratio, the fraction of the selected dataset to run for the eval.\n",
+    "\n",
+    "**Warning**: Many datasets contain thousands of examples which can lead to high endpoint usage costs. We advise starting with a small sample ratio (e.g., 1%) to verify the pipeline and then increasing the ratio if desired. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ce3f8a4-ce03-462b-89fa-19cac827c30c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# AzureML settings - please fill in your values\n",
+    "subscription_id = \"<Azure subscription ID>\"\n",
+    "resource_group = \"<Resource group>\"\n",
+    "workspace_name = \"<Workspace name\"\n",
+    "experiment_name = \"<Experiment name>\"\n",
+    "compute_name = \"<AzureML Compute name>\"\n",
+    "\n",
+    "# Eval to run - you can change this to any of the 12 supported eval names\n",
+    "eval_name = \"mmlu_humanities\"\n",
+    "\n",
+    "# Bedrock URL - defaults to Claude 2.1 in us-east-1\n",
+    "bedrock_endpoint_url = (\n",
+    "    \"https://bedrock-runtime.us-east-1.amazonaws.com/model/anthropic.claude-v2:1/invoke\"\n",
+    ")\n",
+    "\n",
+    "# Name of the connection in your Workspace storing AWS access keys\n",
+    "connection_name = \"<Connection name>\"\n",
+    "\n",
+    "# Sample ratio - what fraction of the dataset to run for the eval?\n",
+    "# **WARNING** be aware of endpoint costs!\n",
+    "sample_ratio = 0.01"
```

Comment:
Should we set it to 0.01 by default or set it to 1.0 and in comment, recommend adjusting to 0.01 for testing e2e works and then switching back to 1.0?

Reply:
Would prefer to leave the default at 0.01 so users aren't surprised by long runtimes and high costs :)